resources/ips/doc-transform.erb

```diff
@@ -0,0 +1,2 @@
+<transform dir path=<%= pathdir %>$ -> edit group bin sys>
+<transform file depend -> edit pkg.debug.depend.file ruby env>
```

Comment:
What does this dependency represent? Would it be valid in a non Ruby project? Does it somehow create a dependency on a Ruby outside of the Omnibus package we are creating?